app/models/ems_refresh.rb

```diff
@@ -14,6 +14,7 @@ module EmsRefresh
   extend EmsRefresh::SaveInventoryConfiguration
   extend EmsRefresh::SaveInventoryAutomation
   extend EmsRefresh::SaveInventoryOrchestrationStacks
+  extend EmsRefresh::SaveInventoryPhysicalInfra
```

Comment:
This line is already in master https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_refresh.rb#L6